modules/mri_violations/php/NDB_Menu_Filter_Form_resolved_violations.class.inc

```diff
@@ -134,13 +134,22 @@ class NDB_Menu_Filter_Form_resolved_violations extends NDB_Menu_Filter_Form
     {
         // set the class variables
         $config =& NDB_Config::singleton();
+        // create user object
+        $user =& User::singleton();
         $useProjects = $config->getSetting("useProjects");
         if ($useProjects === "false") {
             $useProjects = false;
         }
         else{
             $useProjects = true;
         }
+        // Get sites
+        $sitesCond = '';
+        if (!$user->hasPermission('access_all_profiles')
+            && !is_null($user->getData('CenterID'))
+        ) {
+            $sitesCond = " AND p.CenterID = '".$user->getData('CenterID')."' ";
```

Comment:
Remark: p.CenterID is a numeric column and not text. The ' characters should be removed.
But MySQL is taking care of it.